sr_Latn_RS/webapp/src/main/webapp/i18n/vivo_all_sr_Latn_RS.properties

```diff
@@ -0,0 +1,1003 @@
+
+select_existing_pub_or_enter_new = Molimo Vas da odaberete postojeću publikaciju u polju za Naslov ili unesite novu.
+
+publication_entry_for = unos publikacije za
+publication_type = Tip publikacije
+selected_publication = Odabrana publikacija
+published_in = Objavljeno u
+selected_journal = Odabran časopis
+selected_event = Odabran događaj
+# Need context
+proceedings_of = Obrada
+editor_capitalized = Urednik
+#TODO need context
+required_with_last_name = Zahtevano uz novo prezime
+selected_editor = Odabran urednik
+publisher_capitalized = Izdavač
+selected_publisher = Odabran izdavač
+#TODO Not sure, maybe just to translate it as "Knjiga"
+volume_capitalized = Tom
+number_capitalized = Broj
+issue_capitalized = Izdanje
+chapter_capitalized = Poglavlje
+start_page = Početna strana
+end_page = Završna strana
+selected_book = Odabrana knjiga
+publication_date = Datum objavljivanja
+place_of_publication = Mesto objavljivanja
+
+research_activity = aktivnosti istraživanja
+research_activity_type = tip aktivnosti istraživanja
+
+reviewer_of = recenzent za
+
+#Need Context
+entry_for = unos za
+specify_role_for_activity = Molimo Vas unesite ulogu za ovu aktivnost.
+start_year = Početna godina
+
+#TODO Not sure
+service_to_profession = služenje struci
+service_to_profession_in = služenje struci u
+
+teaching_activity = nastavne aktivnosti
+teaching_activity_type = tip nastavne aktivnosti
+teaching_role_hint = npr. instruktor, profesor, asistent
+
+create_own_concept_all_caps = Kreirajte Vaš koncept
+concept_capitalized = Koncept
+selected_concept = Odabran koncept
+create_concept = Kreirajte koncept
+return_to_manage_concepts = Povratak na upravljanje konceptima
+
+#TODO not sure
+institutional_internal_class = Institucionalna unutrašnja klasa
+internal_class_intro_one = Ova klasa će se koristiti za određivanje pojedinaca unutar Vaše institucije.
+internal_class_intro_two = Ovo će Vam omogućiti da ograničite (filtrirate) pojedince koji se prikazuju na Vašim meni-stranicama (Ljudi, Istraživanja, itd.) na samo one koji su deo vaše institucije.
+no_local_oncologies = Trenutno nema prepoznatljivih lokalnih ontologija.
+namespace_must_use_this_pattern = Da bi lokalna ontologija ovde bila prepoznata, URI njenog imenskog prostora (namespace URI) mora da prati sledeći šablon
+new_local_oncology = nova lokalna ontologija
+return_here_to_define_class = i zatim se vratite ovde da definišete unutrašnje klase institucije.
+#TODO not sure, need context
+select_existing_local_class = Odaberite postojeću klasu iz lokalnog proširenja
+cannot_find_class = Ne možete da pronađete odgovarajuću klasu?
+create_new_class = Kreirajte novu klasu
+create_new_one = Kreirajte novu
+use_capitals_each_word = koristite veliko slovo na početku svake reči
+local_namespace = Lokalni prostor imena (Namespace)
+#TODO Dont understand what this sentence means
+problematic_section_error = Greška: problematična sekcija, kao što je gore navedena, je trebalo da budu rešene.

```

Comment:
Greška: problematična sekcija, kao što je gore navedena, je trebalo već da bude rešena.